### PR TITLE
[DW-49]: Add new input to process activiti-filters component to show/hide icons

### PR DESCRIPTION
**Pull request by infad-kachancheri**
added hasIcon input to hide/show filter icons.

### CODE REVIEW

File: ng2-components/ng2-activiti-processlist/src/components/activiti-filters.component.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
         <li class="mdl-list__item activiti-filters__entry" (click)="selectFilter(filter)" *ngFor="let filter of filters"
             [class.active]="currentFilter === filter">
             <span class="mdl-list__item-primary-content">
-            <i class="material-icons mdl-list__item-icon activiti-filters__entry-icon" [attr.data-automation-id]="filter.name + '_filter'" >assignment</i>
+            <i *ngIf="hasIcon" class="material-icons mdl-list__item-icon activiti-filters__entry-icon" [attr.data-automation-id]="filter.name + '_filter'" >assignment</i>
```
Comment (eromano) [inline ng2-components/ng2-activiti-processlist/src/components/activiti-filters.component.html]: Thanks for the PR, I think in this case a better verb to use should be show for reflecting the functionality

Comment (infad-kachancheri) [inline ng2-components/ng2-activiti-processlist/src/components/activiti-filters.component.html]: Hi @eromano ,

"hasIcon" we got it from ng2-activiti-tasklist/src/components/activiti-filters.component.html. AD F is using this verb in tasklist filter component thatswhy we also followed same. Can you please confirm still we need to change this or not.

Comment (eromano) [inline ng2-components/ng2-activiti-processlist/src/components/activiti-filters.component.html]: I think also the other is wrong should be show

Comment (infad-kachancheri) [inline ng2-components/ng2-activiti-processlist/src/components/activiti-filters.component.html]: Can I use "showIcon" instead of "hasIcon"?

Comment (eromano) [inline ng2-components/ng2-activiti-processlist/src/components/activiti-filters.component.html]: yep

Comment (infad-kachancheri) [inline ng2-components/ng2-activiti-processlist/src/components/activiti-filters.component.html]: "hasIcon" Input changed to "showIcon"

**Pull request merged**